src/jobs/utils/update_cards.ts

```diff
@@ -0,0 +1,111 @@
+import { ART_SERIES_CARD_SUFFIX } from '../../client/utils/cardutil';
```

Comment:
Happy to adjust the file name / path if there is a preference for something else